package.json

```diff
@@ -43,7 +43,7 @@
     "linux": {
       "category": "Network;Chat",
       "packageCategory": "net",
-      "executableName": "teams",
+      "executableName": "teams-for-linux",
```

Comment:
It seems a reasonable change but, if someone is using the other executable name, they might complain a bit... 

If deciding to got ahead, change also the reference in this readme (https://github.com/IsmaelMartinez/teams-for-linux/blob/master/app/config/README.md)

Reply:
I think this is a logical change, despite there being some inertia here. I'd vote for a clearly noted readme/changelog entry noting the change.